OpenWeatherTests/ViewModels/HomeViewModelTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OpenWeather template generated by OpenBytes on 02/04/2023.
+//
+// Created by Ahmed Shendy.
+//  HomeViewModelTests.swift
+//
+
+import XCTest
+import Test
+@testable import OpenWeather
+
+final class HomeViewModelTests: XCTestCase {
+
+    func testLondonLocationWeather() async throws {
+        let sut: HomeViewModel = .mock
+
+        sut.getWeather(for: .london)
+
+        let locationName = "London"
+        let temperature = "\(Mock.londonWeather.currentTemperature.abbreviatedAsProvided)"
+        let symbolName = "\(Mock.londonWeather.symbolName)"
+        let realFeel = "\(Mock.londonWeather.realFeel.abbreviatedAsProvided)"
+        let uv = "\(Mock.londonWeather.uv)"
+        let windSpeed = "\(Mock.londonWeather.wind.speed.abbreviatedAsProvided)"
+
+        try await Waiter(sut).wait(for: \.value.content) { content in
+            content.locationName == locationName
+            && content.temperature == temperature
+            && content.symbolName == symbolName
+            && content.realFeel == realFeel
+            && content.uv == uv
+            && content.windSpeed == windSpeed
+        }
```

Comment:
We could create our own `expectedWeather` object and just make `Weather` equatable. This is fine, but the multiple && make it verbose.

Also creating an actual Weather object, we will be forced to update these tests if that model changes... Here we don't actual create one so we don't get an error if there is a new value added.